mobile/src/feeds/feed.tsx

```diff
@@ -19,8 +19,8 @@ interface FeedListItemState {
 }
 
 export class FeedListItem extends React.Component<FeedListItemProps, FeedListItemState> {
-    constructor(props: FeedListItemProps, context: any) {
-        super(props, context);
+    constructor(props: FeedListItemProps) {
+        super(props);
```

Comment:
why did you make this change? Don't you lose a passed argument in this case?

Reply:
Because
https://github.com/SoftwareCountry/arcadian-assistant/pull/37#pullrequestreview-93824242